frontend/src/components/models.ts

```diff
@@ -13,9 +13,47 @@ export interface MulticallResponse {
   returnData: string[];
 }
 
+export type Chain = {
+  chainId: string;
+  chainName: string;
+  nativeCurrency: {
+    name: string;
+    symbol: string;
+    decimals: 18;
+  };
+  rpcUrls: string[];
+  blockExplorerUrls?: string[];
```

Comment:
Would be great if the `blockExplorerUrl: string` field was directly used within the `getEtherscanUrl()` method in the `utils.ts` file! That way when adding a network we can just do it all in this file